src/protocols/http.js

```diff
@@ -78,25 +41,7 @@ class HttpWrapper extends KuzzleAbstractProtocol {
     }
 
     return this._sendHttpRequest('GET', '/')
```

Comment:
This is not used anymore, you should remove this call because it may throw an error if the anonymous user does not have the server:info right

Reply:
Anonymous can use the HTTP protocol since this function will not reject in case of permissions issue. You'll still get a response from the server (with 403 status code yes but a response). This will rejects only in case of network issue. :slightly_smiling_face: